### ISSUE TITLE
[BUG] Set-PnPListItem: A parameter cannot be found that matches parameter name 'Batch'.

### ISSUE THREAD

**Issue by achimismaili** [pnp/powershell#402]
### Reporting an Issue 
Cannot use batching for command Set-PnPListItem in an image library

### Expected behavior 
Expected to set items in batch mode with provided code snippets

### Actual behavior 
Error message in PowerShell when running command:
**Set-PnPListItem: A parameter cannot be found that matches parameter name 'Batch'.**

### Steps to reproduce behavior
Please include complete script or code samples in-line or linked from [gists](https://gist.github.com/)

Add a picture library with a renamed thumbnail and a location column:
```
Enable-PnPFeature -Identity "6e1e5426-2ebd-4871-8027-c5ca86371ead" -Scope Site
New-PnPList -Title $Url -Url $Url -Template PictureLibrary -EnableVersioning -OnQuickLaunch
Add-PnPField -List $lib -DisplayName "Thumbnail" -InternalName "Thumbnail" -Type "Thumbnail" -AddToDefaultView -AddToAllContentTypes
Set-PnPField -Identity $thumbnail.Id -Values @{Title = "Bildvorschau" } -List $lib
Add-PnPField -List $lib -DisplayName "Ort" -InternalName "Location" -Type "Location" -AddToDefaultView -AddToAllContentTypes
```

Then, when trying to update documents within the library, the batch parameter is not known.
Trying to use Set-PnPListItem with batch mode, but parameter seems not to be known.
Goal is to set an added location type column in a image library

```
$batch = New-PnPBatch
    foreach ($item in $ltems) {
        Write-Output "$($item.FileLeafRef) with ID $($item.Id)"
        Set-PnPListItem -List "Fotos" -Identity $item.Id -Batch $batch -Values @{"Location"=$item.Location} 
    }
Invoke-PnPBatch -Batch $batch    
```

It did not help to set the -Batch parameter at the end and it did also not help to try to set the title instead of the location.
When not using the "-Batch" parameter, it works fine

### What is the version of the Cmdlet module you are running?
1.3.0

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment]
@achimismaili - this has been fixed in the nightly releases.
Can you try the prerelease version - `Install-Module -Name PnP.PowerShell -AllowPrerelease` and check ?
If you can't use prerelease version, you need to wait for 1.4 release which will be coming in a couple of weeks

**Comment by gautamdsheth** [comment]
Closing it since this parameter is available in 1.4 which has been released now.
